### DISCUSSION TITLE
inline logs location

### DISCUSSION THREAD

**Discussion by mohan-nagandlla** [strimzi/strimzi-kafka-operator#5760] [Q&A]
**Suggestion / Problem**
we can able to use log4j.properties to capture the logs by inline or external methods external method is clear but the probelm is at inline method where will the logs will store? where we can see the all logs files.

**Documentation Link**
https://strimzi.io/docs/operators/latest/using.html#property-kafka-connect-logging-reference

**Comment by scholzj** [comment]
As common in Kubernetes, the logs go to standard output where you can collect them with your container engine and tools such as Fluentd and similar. Please feel free to open a PR if you think this needs to be clarified in the documentation.

**Comment by mohan-nagandlla** [comment]
Thank you and isn't possible to capture the connect logs inside Kafka broker

**Comment by scholzj** [comment]
WDYM with _capture the connect logs inside Kafka broker_? As in send them to Kafka as Kafka messages?
